payments/management/commands/send_sap_invoices.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import paramiko
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.utils import timezone
+
+from payments.models import Invoice
+
+LOG = logging.getLogger(__name__)
+
+SFTP_SERVER_IP = settings.RESPA_INVOICES_SFTP_SERVER_IP
+SFTP_USERNAME = settings.RESPA_INVOICES_SFTP_USERNAME
+RSA_KEY_PATH = settings.RESPA_INVOICES_SFTP_RSA_KEY_PATH
+RSA_KEY_PASSWORD = settings.RESPA_INVOICES_SFTP_RSA_KEY_PASSWORD
+
+
+class Command(BaseCommand):
+    help = "Sends the SAP invoices as XML files to an SFTP server."
+
+    def handle(self, *args, **options):
+        invoices = Invoice.objects.filter(
+            xml_generated_at__isnull=False, sent_to_sap_at__isnull=True
+        )
+
+        upload_xml_to_sftp(invoices)
+
+
+def upload_xml_to_sftp(invoices):
+    """
+    Establishes a connection to the SFTP server and
+    uploads the SAP invoices there as XML files.
+
+    :param invoices The invoices to send to SAP
+    :type invoices Queryset/List of Invoice objects
+    """
+
+    # File path on the SFTP server
+    remote_path = "/out/"
```

Comment:
Remote path should probably be passed as an argument to the command rather than hard coded

Reply:
Or just have "/out/" as the default value for the argument

Reply:
That sounds a good idea, actually.

Reply:
Added the remote path as an argument to the command and set the default to "/out"/.